.github/workflows/package-mdk-bindings.yml

```
@@ -25,7 +25,14 @@ env:
 
 jobs:
   package-swift:
-    runs-on: macos-latest
+    # Pin to macos-26-arm64 so the iOS SDK we build against matches what
+    # downstream consumers on Xcode 26 expect. macos-latest currently resolves
+    # to macos-15-arm64 whose default Xcode is 16.x (iPhoneSimulator 18.5 SDK),
+    # and the resulting sim slice — while structurally a valid iossimulator
+    # build — mixes sdk=18.5 and sdk=26.x objects, which Xcode 26's stricter
+    # linker rejects with the platform-mismatch error reported in
+    # marmot-protocol/mdk-swift#1.
+    runs-on: macos-26-arm64
```

actionlint (1.7.12)

[error] label "macos-26-arm64" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)